build/common.sh

```diff
@@ -553,7 +553,7 @@ setup_kernel()
 {
 	echo ">>> Setting up kernel in ${1}"
 
-	tar -C ${1} -xpf ${SETSDIR}/kernel-*-${PRODUCT_ARCH}.txz
+	tar -C ${1} -xpf ${SETSDIR}/kernel-*-${PRODUCT_ARCH}-${PRODUCT_DEVICE}.txz
```

Comment:
can't do this right now without breaking production upgrades

Reply:
Can review latest commit?